Makefile.am

```diff
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4
 AUTOMAKE_OPTIONS = foreign
-SUBDIRS = src
+SUBDIRS = src test
```

Comment:
Hmmm, I believe it should be a configure option to build tests, as is the case with the Lua wrapper.